example/app/config.yml

```diff
@@ -423,6 +427,18 @@ jl:
     shared-2:
       name: rvl-internal
       type: shared
+  ecs:
+     ssh: true
+     keyPairName: jl
+  ems:
```

Comment:
Looks like this configuration needs to move up so we can deploy it. We also need documentation on all of this.

I think we need the ems configuration to be optional so the deployment doesn't fail if it's not there.

Reply:
`ems:` configuration has default value in @cumulus/deployment/app/config.yml, so it's optional if the operator doesn't specify it.